suites/squid/common/regression/1_ss_ssl_rgw.yaml

```diff
@@ -0,0 +1,420 @@
+# Conf file : conf/squid/common/5node_1client_3rgw_ss.yaml
+
+tests:
+  - test:
+      abort-on-fail: true
+      desc: Install software pre-requisites for cluster deployment.
+      module: install_prereq.py
+      name: setup pre-requisites
+
+  - test:
+      abort-on-fail: true
```

Comment:
Bootstrap will be part of deployment, add steps to configure ssl or non-ssl rgw daemons